### DISCUSSION TITLE
Browse mode on media quality

### DISCUSSION THREAD

**Discussion by shfinne** [LMS-Community/slimserver#791] [Ideas]
I have my music in different formats audiobooks in mp3 most music in flac 16/44.1 but there is a growing amount of HD stuff 24/88.2-192 and then there is DSD. Well I have not found a way to Browse only the HD stuff.  Gender is not a good way like done with the audiobooks An easy way can be to be able to define a lib that only browse on a specific path(s). That will solve it in what a guess most will be able to live with. 

Styrk

**Comment by michaelherger** [comment]
This should be possible using library views. Use the advanced search in the web UI, search for the bitrates. Once you got the results you want, save the search as a library view.

It's a common topic and well covered on https://forums.slimdevices.com. I'd suggest you join the community there. There are many discussions around how to create library views.
